src/fides/config/config_proxy.py

```diff
@@ -54,8 +54,27 @@ class ExecutionSettingsProxy(ConfigProxyBase):
     prefix = "execution"
 
     subject_identity_verification_required: bool
+    disable_consent_identity_verification: bool
     require_manual_request_approval: bool
 
+    def __getattribute__(self, name: str) -> Any:
+        """
+        Overrides base __getattribute__ to provide a dynamic fallback for
+        'disable_consent_identity_verification'. The fallback is based on
+        'subject_identity_verification_required' only when no explicit value is provided,
+        preserving None for unset cases.
+        """
+        if name == "disable_consent_identity_verification":
+            value = super().__getattribute__("disable_consent_identity_verification")
+            if value is None:
+                subject_verification_required = super().__getattribute__(
+                    "subject_identity_verification_required"
+                )
+                return not subject_verification_required
+            return value
+
+        return super().__getattribute__(name)
+
```

Comment:
That's correct! This was a tricky one that I got some help from Adam for. I originally had this fallback logic in the `ExecutionSettings` class but that didn't account for cases where `subject_identity_verification_required` was updated through the API. The behavior that I wanted was that as long as `disable_consent_identity_verification` isn't defined, it will continue to be derived from `subject_identity_verification_required`. Once a user sets a value (not `None`) then we will stop performing the fallback. The only way to get this to work is if we derive the fallback in the `ExecutionSettingsProxy` class.